packages/web-app-password-protected-folders/README.md

```diff
@@ -10,11 +12,52 @@ This web extension enhances the oCIS platform by allowing users to create passwo
+   1. If you create a PPF in a project Space where users also have delete permissions, **Any** accessing user with delete permission would be able to delete the link file. Accidentially or intentionally done, no one would be possible to access the PPF anymore though the data of the target folder still exists.
+   1. As suggestion for a secure environment accessing PPF's, it is recommended to create an own project Space only for PPF's where added users only have view permissions. Space managers or defined users can or will have extended permissions though. This guarantees that users that are allowed accessing the Space wont be able to delete or manipulate link files "accidentially".  
+
+1. **Restoring a Deleted a PPF**
+
+   It is most important to understand that resources are not accessed by their name but their internal UNIQUE ID (UUID).
+
+   1. If a PPF got deleted by the creator, both the link file and the target folder get moved to the trashbin of their respective Space. Though you can restore both, the PPF will no longer work as the underlaying UUID has changed.
+   1. To recreate a deleted PPF (with the same name), you must exactly follow the described steps below:
+      1. If not already done, the PPF creator needs to enable showing hidden files in his private Space.
+      1. Undelete the target folder from the trashbin.
+      1. Rename the restored target folder, the name MUST NOT be the same as it was before.
+      1. DO NOT undelete the link file from the project Space!! This file can finally be deleted.
+      1. Create a new PPF. You can now use the former PPF name because the original target folder restored has a different name now.
+      1. Copy or move any data from the restored and renamed target folder into the fresh created target folder.
+      1. Delete the renamed target folder, it is not needed any more.
+      1. If wanted, prevent showing hidden files in the personal Space again and prevent them from being viewed.
+
+1. **Moving or Renaming a Link File**
+
+   Moving or renaming a link file is prevented by the webUI, but you could copy this file or manipulate it via the webUI if permission allows or via one of the Desktop, iOS or Android apps if `.psec` files are not excluded from the sync list. Some things need to be considered for this case:
+   
+   1. Link files and the target folder share the same path. When the owner deletes a moved, renamed or copied link file, it will remove the link file but not the target folder because it does not have a valid mirrored path anymore to the target folder and cant find it. This operation fails silently and nothing gets reported.
+   1. A deletion of a manipulated link file requires a manual deletion of the target folder. If this step is ommitted,  it can be a cause of orphaned data!!
```

Comment:
```suggestion
   1. A deletion of a manipulated link file requires a manual deletion of the target folder. If this step is ommitted, it can be a cause of orphaned data!!
```